challenges/qr_code_component/index.html

```diff
@@ -17,46 +17,71 @@
       rel="icon"
       type="image/png"
       sizes="32x32"
-      href="../images/favicon-32x32.png"
+      href="images/favicon-32x32.png"
```

Comment:
No need for this change. You can also remove that favicon from the qr_code_component/images directory.